mobile/src/main/java/ch/epfl/sdp/mobile/ui/setting/Emojis.kt

```diff
@@ -0,0 +1,3 @@
+package ch.epfl.sdp.mobile.ui.setting
+
+val emojis = listOf("😎", "🐟", "👽", "🎧", "😂")
```

Comment:
By convention, top-level `val` are named in PascalCase. Also, you should add some documentation.

```kotlin
val Emojis = listOf("😎", "🐟", "👽", "🎧", "😂")
```

Reply:
is changed